File: pennyway-socket/src/main/java/kr/co/pennyway/socket/service/LastMessageIdSaveService.java
package kr.co.pennyway.socket.service;

import kr.co.pennyway.domain.context.chat.service.ChatMessageStatusService;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@Service
@RequiredArgsConstructor
public class LastMessageIdSaveService {
    private final ChatMessageStatusService chatMessageStatusService;

    public void execute(Long userId, Long chatRoomId, Long lastReadMessageId) {
        chatMessageStatusService.saveLastReadMessageId(userId, chatRoomId, lastReadMessageId);
    }
}

File: pennyway-socket/src/main/java/kr/co/pennyway/socket/service/LastMessageIdSaveService.kt
package kr.co.pennyway.socket.service;

import kr.co.pennyway.domain.context.chat.service.ChatMessageStatusService
import org.springframework.stereotype.Service

@Service
class LastMessageIdSaveService(private val chatMessageStatusService: ChatMessageStatusService) {
    fun execute(userId: Long, chatRoomId: Long, lastReadMessageId: Long) {
        chatMessageStatusService.saveLastReadMessageId(userId, chatRoomId, lastReadMessageId)
    }
}
